generate/jsonFiles.js

```diff
@@ -0,0 +1,29 @@
+import path from 'path'
+import {writeJsonArray, writeJsonObject} from './writeToJsonFile'
```

Comment:
Oh, sorry, I totally forgot to mention, could you name all files with `kebab-case` rather than `camelCase` to avoid issues with `git` file renames and other cross-platform problems?